template.typ

```diff
@@ -58,14 +57,13 @@
     align(
       center,
       [
-
         #image(logo, width: 6.5cm)
         #v(2em, weak: true)
 
-        #upper[Dipartimento di #text(department)]
+        #upper[#i18n.at(locale).department_of #text(department)]
         #v(2em, weak: true)
 
-        #text(size: 14pt, strong([Corso di #text(course)]))
+        #text(size: 14pt, strong([#i18n.at(locale).degree_course_in #text(course)]))
```

Comment:
Same thing as before, "Computer Engineering Degree" instead of "Degree programme class in"

Reply:
same for this